crawler.py

```diff
@@ -265,3 +266,62 @@ def run_all(driver, data_folder):
 
     for filename in ['code-coverage-gcda.zip', jsvm_output_file]:
         os.remove(filename)
+
+    with open('{}/diff.json'.format(data_folder), 'r') as report:
```

Comment:
Let's move all the code to generate the HTML report into a separate Python module.